test/Elements.Tests/VectorTests.cs

```diff
@@ -84,34 +85,206 @@ public void DistanceToPlane()
+                new Vector3(1.85770976909928,3.12588802475747,0),
+                new Vector3(3.72227455579672,2.49522640572745,0),
+                new Vector3(1.99481012106233,0.562111443048493,0),
+                new Vector3(-0.500416284665121,-0.411301055889139,0),
+                new Vector3(2.99564269039257,0.534691372655883,0),
+                new Vector3(2.88596240882213,1.1653529916859,0),
+                new Vector3(3.79082473177824,1.42584366041569,0),
+                new Vector3(4.33922613963043,-1.08309278050807,0),
+                new Vector3(0.664936707020777,-2.82426725043876,0),
+                new Vector3(3.16016311274823,-2.89281742642029,0),
+                new Vector3(3.76340466138563,-2.13876549062353,0),
+                new Vector3(4.13357561168586,-3.00249770799072,0),
+                new Vector3(2.22788071939951,-3.72912957339487,0),
+                new Vector3(-1.3093083612471,-3.11217798956116,0),
+                new Vector3(1.65205924115471,-1.49439383639721,0),
+                new Vector3(-2.43353124734408,-1.67262429394917,0),
+                new Vector3(-3.24242332392606,-3.20814823593529,0),
+                new Vector3(-3.98276522452651,-2.27586584258658,0),
+                new Vector3(-2.85854233842953,-0.671791724618928,0)
+            });
+            var voids = new Polygon[]
+            {
+                new Polygon(new []
+                {
+                    new Vector3(2.95451258480366,-0.82260211177828,0),
+                    new Vector3(2.83210175913191,-0.184080777868908,0),
+                    new Vector3(0.18606496624511,-0.691352080132181,0),
+                    new Vector3(0.308475791916855,-1.32987341404155,0)
+                }),
+                new Polygon(new []
+                {
+                    new Vector3(-2.44724128254038,-0.671791724618928,0),
+                    new Vector3(-1.86692577850678,-0.130163920854226,0),
+                    new Vector3(-2.25080676400331,0.281137135034914,0),
+                    new Vector3(-2.83112226803692,-0.260490668729788,0)
+                }),
+                new Polygon(new []
+                {
+                    new Vector3(2.37869110655886,1.63149418836026,0),
+                    new Vector3(2.37869110655886,2.6323267576905,0),
+                    new Vector3(1.93996998027711,2.6323267576905,0),
+                    new Vector3(1.93996998027711,1.63149418836026,0)
+                }),
+                new Polygon(new []
+                {
+                    new Vector3(-3.28355342951497,-2.6460367928868,0),
+                    new Vector3(-2.87188183213404,-2.34663926751885,0),
+                    new Vector3(-3.20092267684535,-1.8942081060408,0),
+                    new Vector3(-3.61259427422628,-2.19360563140875,0)
+                })
+            };
+
+            var profile = new Profile(perimeter, voids, default, "");
+            var ptsToTest = new Dictionary<Vector3, bool>
```

Comment:
This seems like a really specific dictionary of points. Do we want to put these in a JSON and read them in? What are they?

Reply:
It's just a test of a crazy polygon I drew in rhino along with a hundred random samples and their results. Just making sure our containment results agree with theirs. But I can see how this is overkill.